### ISSUE TITLE
Make hello world message

### ISSUE THREAD

**Issue by mckelveygreg** [OpenEugene/openboard#33]
Back:
@daved, What does the hello world message look like to an outsider?

Front:
@nqthqn, what needs to be set up in Elm if we are to communicate with @daved 's [hello world message branch](https://github.com/champagneabuelo/openboard/tree/tut/hello_service)?

**Comment by nqthqn** [comment]
You would make a command and then ask Elm to run it for you -

```elm
import Http

type Msg
  = GotHello (Result Http.Error String)

getHello : Cmd Msg
getHello =
  Http.get
    { url = "/hello"
    , expect = Http.expectString GotHello
    }
```

Note - not sure on the `url` and instead of `String` you would use a generated type alias and decoder from an `Entity/` module. 

To run the command,

```elm
init = 
    (model, getHello)
```

To react to the response from the command:

```elm
update msg model =
    case msg of
        GotHello (Ok response) ->
            ({ model | hello = response }, Cmd.none)

        GotHello (Err _) ->
            ({ model | hello = "uh oh" }, Cmd.none)
```

You could of course pattern match on the `Http.Error` and give the user more than just "uh oh".

**Comment by daved** [comment]
Closed in favor of #76
